dmoj/checkers/easy.py

```diff
@@ -4,7 +4,7 @@
 from dmoj.utils.unicode import utf8bytes
 
 
-def check(process_output, judge_output, Counter=Counter, regex=re.compile(br'\s+'), **kwargs):
+def check(process_output: str, judge_output: str, Counter=Counter, regex=re.compile(br'\s+'), **kwargs) -> Counter:
```

Comment:
All these checkers, though they work with given `str` inputs, should be `bytes`. Any callers not calling them with `bytes` are buggy, and should be flagged by the typechecker.